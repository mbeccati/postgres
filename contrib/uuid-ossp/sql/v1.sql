
SELECT uuid_generate_v1() < uuid_generate_v1();
SELECT uuid_generate_v1() < uuid_generate_v1mc();

SELECT substr(uuid_generate_v1()::text, 20) = substr(uuid_generate_v1()::text, 20);
SELECT substr(uuid_generate_v1()::text, 20) <> substr(uuid_generate_v1mc()::text, 20);
SELECT substr(uuid_generate_v1mc()::text, 20) <> substr(uuid_generate_v1mc()::text, 20);



