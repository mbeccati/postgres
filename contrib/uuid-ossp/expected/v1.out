SELECT uuid_generate_v1() < uuid_generate_v1();
 ?column? 
----------
 t
(1 row)

SELECT uuid_generate_v1() < uuid_generate_v1mc();
 ?column? 
----------
 t
(1 row)

SELECT substr(uuid_generate_v1()::text, 20) = substr(uuid_generate_v1()::text, 20);
 ?column? 
----------
 t
(1 row)

SELECT substr(uuid_generate_v1()::text, 20) <> substr(uuid_generate_v1mc()::text, 20);
 ?column? 
----------
 t
(1 row)

SELECT substr(uuid_generate_v1mc()::text, 20) <> substr(uuid_generate_v1mc()::text, 20);
 ?column? 
----------
 t
(1 row)

